<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aksesuarSatisTable</class>
 <widget class="QDialog" name="aksesuarSatisTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aksesuar Satış Tablosu</string>
  </property>
  <widget class="QTableWidget" name="tbl_AksesuarSatis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_anamenu">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>AnaMenu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ara">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ara">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ara</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
